corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -186,12 +172,6 @@ <h2>{% trans "Manage Organization Structure" %}</h2>
 
 </div>
 
-
-<link href="{% static 'select2/dist/css/select2.min.css' %}" rel="stylesheet"/>
-<script src="{% static 'select2/dist/js/select2.full.min.js' %}"></script>
```

Comment:
just confirming you removed these two lines because you added the `use_select2_v4` decorator to the view, is that right?

Reply:
yep